content/en/docs/application-store/external-apps/deploy-gitlab.md

```diff
@@ -0,0 +1,153 @@
+
+   ```yaml
+   global:
+       hosts:
+         domain: demo-project.svc.cluster.local
+   
+     gitlab-runner:
+       install: false
+   
+     gitlab:
+       webservice:
+         helmTests:
+           enabled: false
+   ```
+
+   ![change-value](/images/docs/appstore/external-apps/deploy-gitlab/change-value.PNG)
+
+   {{< notice note >}}
+
+   The `demo-project` refers to the project name where GitLab is deployed. Make sure you use your own project name.
+
+   {{</ notice >}}
+
+7. Wait for GitLab to be up and running.
+
+   ![gitlab-running](/images/docs/appstore/external-apps/deploy-gitlab/gitlab-running.PNG)
+
+8. Go to **Workloads**, and you can see all the Deployments and StatefulSets created for GitLab.
+
+   ![deployments-running](/images/docs/appstore/external-apps/deploy-gitlab/deployments-running.PNG)
+
+   ![statefulsets-running](/images/docs/appstore/external-apps/deploy-gitlab/statefulsets-running.PNG)
+
+   {{< notice note >}}
+
+   It may take a while before all the Deployments and StatefulSets are up and running.
+
+   {{</ notice >}}
+
+### Step 3: Get the root user's password
+
+1. Go to **Secrets** under **Configurations**, input `gitlab-initial-root-password` in the search bar, and then press **Enter** on your keyboard to search the Secret.
+
+   ![search-secret](/images/docs/appstore/external-apps/deploy-gitlab/search-secret.PNG)
+
+2. Click the Secret to go to its detail page, and then click the eye icon in the upper right corner to view the password. Make sure you copy it.
+
+   ![click-eye-icon](/images/docs/appstore/external-apps/deploy-gitlab/click-eye-icon.PNG)
+
+   ![password](/images/docs/appstore/external-apps/deploy-gitlab/password.PNG)
+
+### Step 4: Edit the hosts file
+
+1. Find the hosts file on your local machine.
+
+   {{< notice note >}}
+
+   The path of hosts file is `/etc/hosts` for Linux, or `c:\windows\system32\drivers\etc\hosts` for Windows.
+
+   {{</ notice >}}
+
+2. Add the following item into the hosts file.
+
+   ```
+   192.168.4.3  gitlab.demo-project.svc.cluster.local
+   ```
+
+   {{< notice note >}}
+
+   - The `192.168.4.3` and `demo-project` refer to the NodeIP and project name respectively where GitLab is deployed. Make sure you use your own NodeIP and project name.
+   - You can use any IP address of the nodes in your Kubernetes cluster.
+
+   {{</ notice >}}
+
+### Step 5: Access GitLab
+
+1. Go to **Services** under **Application Workloads**, input `nginx-ingress-controller` in the search bar, and then press **Enter** on your keyboard to search the Service. You can see the Service and its type is set to `NodePort` by default and the port `32618` that you can use to access GitLab.
+
+   ![search-service](/images/docs/appstore/external-apps/deploy-gitlab/search-service.PNG)
+
+   {{< notice note >}}
+
+   The port number shown on your console may be different from the port `32618` in this tutorial. Make sure you use your own port number.
```

Comment:
The port number shown on your console may be different from the port `32618` in this tutorial.
->
The port number shown on your console may be different.